Sources/CombineSchedulers/Concurrency.swift

```diff
@@ -76,14 +77,15 @@
       tolerance: SchedulerTimeType.Stride = .zero,
       options: SchedulerOptions? = nil
     ) -> AsyncStream<SchedulerTimeType> {
-      AsyncStream { continuation in
+      @UncheckedSendable var scheduler = self
+      return AsyncStream { [$scheduler] continuation in
         let cancellable = self.schedule(
           after: self.now.advanced(by: interval),
           interval: interval,
           tolerance: tolerance,
           options: options
         ) {
-          continuation.yield(self.now)
+          continuation.yield($scheduler.wrappedValue.now)
```

Comment:
This "fixes" an unrelated issue that cropped up in beta 3. Seems that `self.now` is a detected data race. We could consider a better fix, but this is at least functionally the same as on `main`.